interface/wx/grid.h

```diff
@@ -6346,12 +6346,18 @@ class wxGridSizeEvent : public wxNotifyEvent
     @class wxGridRangeSelectEvent
 
     @beginEventTable{wxGridRangeSelectEvent}
-    @event{EVT_GRID_RANGE_SELECT(func)}
+    @event{EVT_GRID_RANGE_SELECTING(func)}
+        The user is selecting a group of contiguous cells. Processes a
+        @c wxEVT_GRID_RANGE_SELECTING event type.
```

Comment:
```suggestion
        @c wxEVT_GRID_RANGE_SELECTING event type.
        This event is available in wxWidgets 3.1.5 and later only.
```
(and the same below)